multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -27,14 +27,23 @@ type MemberCluster struct {
 	ClusterID string `json:"clusterID,omitempty"`
```

Comment:
In line 26 - "Identify a member cluster in the ClusterSet"

Reply:
In line 24 - "MemberCluster specifies information of a member cluster, which can be a regular member cluster or a leader cluster of the ClusterSet.